aqt/archives.py

```diff
@@ -76,26 +81,26 @@ def __init__(self, version, os_name, target, base, timeout=(5, 5)):
         self._get_archives()
 
     def _get_archives(self):
-        qt_ver_num = self.version.replace(".", "")
-        self.qt_ver_base = self.version[0:1]
         # Get packages index
-        if self.qt_ver_base == "6" and self.target == "android":
+        if self.version.major == 6 and self.target == "android":
             arch_ext = ["_armv7/", "_x86/", "_x86_64/", "_arm64_v8a/"]
         elif (
-            self.qt_ver_base == "5"
-            and int(qt_ver_num) >= 5130
+            self.version.major == 5
+            and self.version >= Version("5.13.0")
```

Comment:
Here can be `self.version in SimpleSpec(">=5.13.0,<6.0")`

Reply:
Sounds like a good use of SimpleSpec to me.